src/EditorFeatures/VisualBasicTest/Diagnostics/MoveToTopOfFile/MoveToTopOfFileTests.vb

```diff
@@ -324,13 +324,13 @@ Module Program
     Sub Main(args As String())
 
     End Sub
-    &lt;[|Assembly:|] Reflection.AssemblyCultureAttribute("de")&gt; 'Bomment
+    &lt;[|Assembly:|] Reflection.AssemblyCultureAttribute("de")&gt; 'Comment
```

Comment:
I wonder if this wasn't deliberate (to distinguish between the other attribute.
Perhaps change to ```'Another Comment```